engine/datapool-collector/src/main/kotlin/io/holunda/camunda/datapool/sender/SimpleDataEntryCommandSender.kt

```diff
@@ -75,10 +78,35 @@ class SimpleDataEntryCommandSender(
 
   override fun sendDataEntryCommand(command: CreateOrUpdateDataEntryCommand) {
     if (properties.enabled) {
-      gateway.send<Any, Any?>(command) { m, r -> logger.debug("Successfully submitted command $m, $r") }
+      gateway.send<Any, Any?>(command) { m, r ->
+        if (r.isExceptional) {
+          errorHandler.apply(m, r)
+        } else {
+          successHandler.apply(m, r)
+        }
+      }
     } else {
       logger.debug("Would have sent command $command")
     }
   }
 }
 
+/**
+ * Success handler, logging.
+ */
+class LoggingCommandSuccessHandler(private val logger: Logger) : DataEntryCommandSuccessHandler {
+
+  override fun apply(commandMessage: Any, commandResultMessage: CommandResultMessage<out Any?>) {
+    logger.debug("Successfully submitted command $commandMessage, $commandResultMessage")
```

Comment:
if `logger` is a standard slf4j logger and not a kotlin-logging one, we should either surround the call with `if(logger.isDebugEnabled())` or use the slf4j placeholder syntax to avoid building strings that will never be logged.